src/java.desktop/macosx/native/libawt_lwawt/java2d/metal/MTLVertexCache.m

```diff
@@ -65,6 +65,9 @@
         MTLVC_ADD_VERTEX(TX1, TY1, DX1, DY1, 0); \
     } while (0)
 
+//Next define should exactly match to the amount of MTLVC_ADD_VERTEX in MTLVC_ADD_TRIANGLES
+#define MTL_TRIS_IN_VERTEX 6
```

Comment:
Constants starting with `MTL_` should be avoided as they are used in Metal Framework library.
I think, this should be `VERTS_FOR_A_QUAD`.